.github/workflows/juno-lint.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.20.3'
+          go-version: '1.21'
```

Comment:
`go-version-file` here? Same for other workflow files.

Reply:
Added to all setup-go actions, thanks!